core/frontend/src/components/vehiclesetup/Configure.vue

```diff
@@ -42,12 +79,19 @@ export interface Item {
 export default Vue.extend({
   name: 'Configure',
   components: {
+    VehicleViewer,
+    VehicleInfo,
+    GripperInfo,
+    PowerInfo,
+    LightsInfo,
+    LeakInfo,
     ParamSets,
     LightsConfigration,
   },
   data() {
     return {
       page_selected: null as string | null,
+      is_dev_mode: settings.is_dev_mode,
```

Comment:
this should probably be a computed property to ensure reactivity

Reply:
Isnt it reactive with this syntax ?

Reply:
nope, seems to need a refresh to take effect when switching pirate mode on/off. same with the name